Source/Details/ASThread.h

```diff
@@ -160,7 +160,7 @@ namespace ASDN {
   public:
 #if !TIME_LOCKER
 
-    Locker (T &l) ASDISPLAYNODE_NOTHROW : _l (l) {
+    Locker (T &l) noexcept : _l (l) {
```

Comment:
I'm not pretty sure if we compile Texture with C++14 at the moment. Can you double check that please. Thanks!

Reply:
Yep this is from `c++11` but glad I double-checked.